cordova-lib/src/cordova/util.js

```diff
@@ -297,7 +297,7 @@ function ensurePlatformOptionsCompatible (platformOptions) {
     }).map(function (arg) {
         return opts[arg] === true ?
             '--' + arg :
-            '--' + arg + '="' + opts[arg].toString() + '"';
+            '--' + arg + '=' + opts[arg].toString();
```

Comment:
I hope so. We're actually already emitting a warning when arguments used this way, so we could try to get rid of this in next major release.